app/src/main/java/net/skyscanner/backpack/demo/compose/CheckboxStory.kt

```diff
@@ -29,18 +29,20 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.state.ToggleableState
-import androidx.compose.ui.tooling.preview.Preview
 import net.skyscanner.backpack.compose.checkbox.BpkCheckbox
 import net.skyscanner.backpack.compose.text.BpkText
 import net.skyscanner.backpack.compose.theme.BpkTheme
 import net.skyscanner.backpack.compose.tokens.BpkSpacing
 import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.CheckboxComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
 
 @Composable
-@Preview
-fun CheckboxStory() {
+@CheckboxComponent
+@ComposeStory
+fun CheckboxStory(modifier: Modifier = Modifier) {
```

Comment:
Also fixed a bunch of stories which missed the modifier – it seems both Android Studio and detekt ignore these cases if story annotated with Preview